### ISSUE TITLE
Centreon-engine does not restart/reload usin the web interface

### ISSUE THREAD

**Issue by iperurena** [centreon/centreon-engine#290]
---------------------------------------------------
BUG REPORT INFORMATION
---------------------------------------------------

**Centreon Web version**: 19.10.1

**Centreon Engine version**: 19.10.5

**Centreon Broker version**:  19.10.5

**OS**: Centos 7.7

**Additional environment details (AWS, VirtualBox, physical, etc.):**
Central poller. 1 server architecture.

**Steps to reproduce the issue:**
1. Remove a host and its associated services
2. Restart/reload Centreon Engine using the web interface.

**Describe the results you received:**
The host and its services are still listed in the monitoring status details. If you click in any of the host's services Centreon says it no longer exists and you should update your configuration.

It seems centreon-engine cannot be restarted using the web interface. After telling it to reload/restart several times I see the uptime of the service does not change. Looking in the centreon-engine configuration I see it references a file (could be the daemon's pidfile?) that is not available in the filesystem. /var/lock/subsys/centengine.lock

**Describe the results you expected:**
The host and its services should no longer be listed after I reload/restart the centreon-engine.
The centreon-engine should be reloadable through the web interface.

**Additional information you think important (e.g. issue happens only occasionally):**
As a temporary workaround I restarted the service using the command-line.

**Comment by lpinsivy** [comment]
Hi,

Can you enable the debug of the Centcore process:

1. Change in /etc/sysconfig/centcore file:
`OPTIONS="--logfile=/var/log/centreon/centcore.log --severity=error --config=/etc/centreon/conf.pm"`
to
`OPTIONS="--logfile=/var/log/centreon/centcore.log --severity=debug --config=/etc/centreon/conf.pm"`
2. In "Administration > Parameters > Debug" menu, enable centcore debug
3. Restart centcore process:
`systemctl restart centcore`

And try to restart your poller and check the log of centcore: 
`tailf /var/log/centreon/centcore.log`

Do you have errors?

**Comment by iperurena** [comment]
Greetings @lpinsivy,
Thank you for getting back to me.

I followed your instructions and saw this in the centcore.log:

`ERROR - Could not write into pipe file /var/lib/centreon-engine/rw/centengine.cmd on poller 1`

Looking in that directory I found a `.centengine.cmd.swp`. Maybe be an ssh session ended abruptly while the file was open and the swp remained there to cause me some  trouble at a later date.

After removing the swp file and undoing the debug changes, I tried reloading and restarting the poller and everything seems to be working fine.
